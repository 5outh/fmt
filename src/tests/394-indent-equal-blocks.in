<?php

$a = $b .
	$c .
	$d;

$a =
	$b .
	$c .
	$d;


function a(){
	$a =
	$b .
	$c .
	$d;
}

$a =
$b =
	$c .
	$d;
