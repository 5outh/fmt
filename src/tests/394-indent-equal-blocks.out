<?php

//example 1
$string = 'a' .
	'b' .
	'c' .
	'd';

//example 2
$string = "a" .
	"b" .
	"c" .
	"d";

//example 3
$a = $b .
	$c .
	$d;

$a =
	$b .
	$c .
	$d;

function a() {
	$a =
		$b .
		$c .
		$d;
}

$a =
$b =
	$c .
	$d;

$a = function ($b) {
	for ($a; $b; $c) {
		doSomething();
	}
	$a = $b .
		$c .
		$d;
};

if ($x = 1) {
	great();
}
$string = 'a' . 'b';

$string = 'a' .
	'b' .
	'c' .
	'd';

$x = ($x = $a);
$x = ($x = $a);

$x = (
	$x = $a
);

$x = $x = $a = $b;

$x = ($x = $a = $b);

if (
	$x = 1
) {
	great();
}

if (
	$x = 1
) {
	great();
}

$string =
	'a' .
	'b' .
	'c' .
	'd';

$x = 1;

$string = [
	"x" => [
		"t" => "1" . $x,
	],
];

if ($x == 1) {
	$string = 'a' .
		'b' .
		'c' .
		'd';

	if ($x = 1) {
		doSomething();
	}

	doSomething($a = 1);
}

while ($row = $percent->fetchObject()) {
	$answers[] = $row;
}

class Test {

	public function abc($x) {
		if (1 == $x) {
			echo "${abc}";
		}
	}

}

Event::trigger("this",
	function () {
		while ($row = $percent->fetchObject()) {
			$answers[] = $row;
		}
	}
);

echo '' .
	implode(
		",\n",
		array_map(
			function ($k, $v) {
				return '`' . $k . '` = ' . $db->quote($v);
			}, array_keys($row), $row
		)
	)
	. '';